sdk/android/app/src/main/assets/skin.json

```diff
@@ -81,7 +81,7 @@
         "color": "white"
       }
     },
-    "showCountDownTimerOnEndScreen": true,
+    "showCountDownTimerOnEndScreen": false,
```

Comment:
you can't change the skin.json as part of the application.  You need to change the javascript code